### ISSUE TITLE
Bug? node debug app.js not serving updated assets

### ISSUE THREAD

**Issue by timestep** [balderdashy/sails#1488]
Seems like when you run node debug app.js, and changes have been made in the assets folder like the javascript/css/html, the changes are not reflected on the site. 

The assets only seems to recompile properly after a sails lift.

Running a restart in node debugger more doesn't do anything either.

**Comment by aabluedragon** [comment]
Happens to me as well, here are a few more details form my dev env:

####Webstorm's output when I run debug (Won't update .tmp form assets [aka the problem]):
```
/usr/local/bin/node --debug-brk=58834 --nolazy /usr/local/lib/node_modules/sails/bin/sails.js lift
debugger listening on port 58834

info: Starting app...

info: debug: 
--------------------------------------------------------info: 
               .-..-.
info: 
debug: :: Tue May 19 2015 23:29:15 GMT+0300 (IDT)
info:    Sails              <|    .-..-.
debug: Environment : development
info: debug: Port        : 1337
debug:    v0.11.0             |\
--------------------------------------------------------info: 
                      /|.\
info:                      / || \
info:                    ,'  |'  \
info:                 .-'.-==|/_--'
info:                 `--'-------' 
info:    __---___--___---___--___---___--___
info:  ____---___--___---___--___---___--___-__
info: 
info: Server lifted in `/Users/alonamir/Work/9veg`
info: To see your app, visit http://localhost:1337
info: To shut down Sails, press <CTRL> + C at any time.
```

####Webstorm's output when I run debug (Updates .tmp from assets like it should):
```
/usr/local/bin/node /usr/local/lib/node_modules/sails/bin/sails.js lift

info: Starting app...

info: 
debug: info:                .-..-.
--------------------------------------------------------info: 
debug: :: Tue May 19 2015 23:31:27 GMT+0300 (IDT)

info: debug: Environment : development
   Sails              <|    .-..-.
info: debug: Port        : 1337
debug:    v0.11.0             |\
--------------------------------------------------------info: 
                      /|.\
info:                      / || \
info:                    ,'  |'  \
info:                 .-'.-==|/_--'
info:                 `--'-------' 
info:    __---___--___---___--___---___--___
info:  ____---___--___---___--___---___--___-__
info: 
info: Server lifted in `/Users/alonamir/Work/9veg`
info: To see your app, visit http://localhost:1337
info: To shut down Sails, press <CTRL> + C at any time.
```

**Comment by japel** [comment]
@aabluedragon I have the same issue
any updates on this?

**Comment by japel** [comment]
As @mikermcneil said, it seems to have something to do with that stupid "debugger listening on <port>" message being piped to stderr.

I managed to get it working by removing any "--debug" or "--debug-brk" arguments from process.execArgv before the child process is created:

```javacript
      var rememberArgs = [];

      for(var i = 0;i<process.execArgv.length; i++){
        if(process.execArgv[i].match(/--debug/)|| process.execArgv[i].match(/--debug-brk/)){
          rememberArgs.push(process.execArgv[i]);
          process.execArgv.splice(i, 1);
        }
      }
```

 @https://github.com/balderdashy/sails/blob/master/lib/hooks/grunt/index.js#L59

and after it is created pushing them back:
```javacript
      for(var j = 0; j<rememberArgs.length;j++){
        process.execArgv.push(rememberArgs[j]);
      }
```

**Comment by japel** [comment]
@sgress454 
may I do a PR for this?

**Comment by japel** [comment]
@sgress454 nevermind... this issue will probably be fixed in 0.12 as stated by you in https://github.com/balderdashy/sails/issues/2670
